app/models/event.rb

```diff
@@ -24,7 +24,7 @@ class Event < ActiveRecord::Base
   include FriendlyId
   friendly_id :name, use: :slugged
 
-  scope :upcoming_events, where('end_date >= ?', Date.tomorrow)
+  scope :upcoming_events, -> { where('end_date >= ?', Time.now) }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.